torch/distributed/algorithms/ddp_comm_hooks/default_hooks.py

```diff
@@ -2,6 +2,18 @@
 import torch.distributed as dist
 
 
+def allreduce_fut(
```

Comment:
Can we add a test ensuring that this allreduce is equivalent to DDP allreduce? Or do you think this is already well covered by current tests?

Reply:
This is more like an internal util function, and both `allreduce_hook` and `powerSGD_hook` are the public APIs that call this util function are tested. I don't think we need a separate test for this util function, as our tests don't need to cover each private/internal function.